lhotse/recipes/aishell2.py

```diff
@@ -0,0 +1,90 @@
+"""
+AISHELL2 (~1000 hours) if available(https://www.aishelltech.com/aishell_2).
```

Comment:
I recommend copying the English description of the dataset from
https://www.aishelltech.com/aishell_2
to here.

Reply:
OK, I will change.